hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/om/ListOpenFilesSubCommand.java

```diff
@@ -168,8 +179,16 @@ private void printOpenKeysList(ListOpenFilesResult res) {
             line += "No\t\t";
           }
         }
+        if (showOverwritten) {
+          if (omKeyInfo.getMetadata().containsKey(OzoneConsts.OVERWRITTEN_HSYNC_KEY)) {
+            line += "Yes\t";
+          } else {
+            line += "No\t";
+          }
+        }
       } else {
         line += showDeleted ? "No\t\tNo\t\t" : "No\t\t";
+        line += showOverwritten ? "No\t" : "";
```

Comment:
```suggestion
        line += showOverwritten ? "No\t\t" : "\t";
```

Reply:
One \t is more aligned, please refer to the above my comment of  "ozone admin om lof --service-host=om --show-deleted --show-overwrittent" output.